requirements/docs.txt

```diff
@@ -3,4 +3,4 @@ sphinxcontrib-httpdomain==1.1.9
 docutils==0.11
 Pygments==1.6
 Sphinx==1.2.2
-mdn-sphinx-theme>=0.3
+sphinx_rtd_theme==0.1.8
```

Comment:
indeed, it's not on https://pyrepo.addons.mozilla.org/
I'll add it straight away though